tests/test_extensions/test_ext_autosummary.py

```diff
@@ -488,6 +488,376 @@ def test_autosummary_generate_content_for_module_imported_members_inherited_modu
+    assert context['module'] == 'autosummary_dummy_complex_inheritance_module'
+    assert context['fullname'] == 'autosummary_dummy_complex_inheritance_module.Jerry'
+
+    assert context['attributes'] == ['relation']
+    assert context['methods'] == [
+        '__init__',
+        'addition',
+        'get_age',
+        'get_name',
+        'get_salary',
+    ]
+
+    assert context['all_attributes'] == [
+        *built_in_attr,
+        'relation',
+    ]
+    assert context['all_methods'] == [
+        '__delattr__',
+        '__dir__',
+        '__eq__',
+        '__format__',
+        '__ge__',
+        '__getattribute__',
+        '__getstate__',
+        '__gt__',
+        '__hash__',
+        '__init__',
+        '__init_subclass__',
+        '__le__',
+        '__lt__',
+        '__ne__',
+        '__new__',
+        '__reduce__',
+        '__reduce_ex__',
+        '__repr__',
+        '__setattr__',
+        '__sizeof__',
+        '__str__',
+        '__subclasshook__',
+        'addition',
+        'get_age',
+        'get_name',
+        'get_salary',
+    ]
+
+    assert context['inherited_attributes'] == [
+        'autosummary_dummy_complex_inheritance_module.Parent.relation'
+    ]
+    assert context['inherited_methods'] == [
+        'autosummary_dummy_complex_inheritance_module.Architect.get_age',
+        'autosummary_dummy_complex_inheritance_module.Job.get_salary',
+        'autosummary_dummy_complex_inheritance_module.Child.addition',
+        'autosummary_dummy_complex_inheritance_module.Child.get_name',
+    ]
+
+    assert context['inherited_qualnames'] == [
+        'autosummary_dummy_complex_inheritance_module.Architect.get_age',
+        'autosummary_dummy_complex_inheritance_module.Job.__dict__',
+        'autosummary_dummy_complex_inheritance_module.Job.__weakref__',
+        'autosummary_dummy_complex_inheritance_module.Job.get_salary',
+        'autosummary_dummy_complex_inheritance_module.Child.addition',
+        'autosummary_dummy_complex_inheritance_module.Child.get_name',
+        'autosummary_dummy_complex_inheritance_module.Parent.relation',
+        'builtins.object.__class__',
+        'builtins.object.__delattr__',
+        'builtins.object.__dir__',
+        'builtins.object.__eq__',
+        'builtins.object.__format__',
+        'builtins.object.__ge__',
+        'builtins.object.__getattribute__',
+        'builtins.object.__getstate__',
+        'builtins.object.__gt__',
+        'builtins.object.__hash__',
+        'builtins.object.__init_subclass__',
+        'builtins.object.__le__',
+        'builtins.object.__lt__',
+        'builtins.object.__ne__',
+        'builtins.object.__new__',
+        'builtins.object.__reduce__',
+        'builtins.object.__reduce_ex__',
+        'builtins.object.__repr__',
+        'builtins.object.__setattr__',
+        'builtins.object.__sizeof__',
+        'builtins.object.__str__',
+        'builtins.object.__subclasshook__',
+    ]
```

Comment:
Can't we generate the `builtins.object.*` methods using `[f'builtins.object.{name}' for name in dir(object)]`?